python/ray/serve/_private/controller.py

```diff
@@ -632,6 +633,28 @@ def deploy_application(self, name: str, deployment_args_list: List[Dict]) -> Non
 
         self.application_state_manager.apply_deployment_args(name, deployment_args_list)
 
+    def deploy_application_xlang(
```

Comment:
we do we need a separate codepath? can we please try to unify them?

Reply:
+1.

Reply:
Okay. The interface for deploying the application has been unified, and the parameters are passed through DeploymentArgsList defined in protobuf.